mlrun/api/api/endpoints/grafana_proxy.py

```diff
@@ -0,0 +1,220 @@
+from http import HTTPStatus
+from typing import List, Dict, Any, Callable
+
+from fastapi import APIRouter, Response, Request
+
+from mlrun.api.api.endpoints.model_endpoints import get_access_key, list_endpoints
+from mlrun.api.schemas import ModelEndpointStateList, GrafanaTable, GrafanaColumn
+from mlrun.config import config
+from mlrun.errors import MLRunBadRequestError
+from mlrun.utils.v3io_clients import get_frames_client
+
+router = APIRouter()
+
+
+@router.get("/grafana-proxy/model-endpoints", status_code=HTTPStatus.OK.value)
```

Comment:
If there's no grafana restriction here, Feels like this should be `/grafana-proxy/health`

Reply:
There is, it requires 2 endpoints:
1) <some-schema>/
2) <some-schema>/query

So i can't use different schemes here